pkg/driver/node/credentialprovider/awsprofile/awsprofiletest/aws_profile.go

```diff
@@ -12,24 +11,15 @@ import (
 	"github.com/awslabs/aws-s3-csi-driver/pkg/util/testutil/assert"
 )
 
-func AssertCredentialsFromAWSProfile(t *testing.T, profileName string, filePerm os.FileMode, configFile, credentialsFile, accessKeyID, secretAccessKey, sessionToken string) {
+func AssertCredentialsFromAWSProfile(t *testing.T, profileName, configFile, credentialsFile, accessKeyID, secretAccessKey, sessionToken string) {
 	t.Helper()
 
-	assertCredentialFilePermissions(t, configFile, filePerm)
```

Comment:
It seems useful to assert that credential files has the correct permissions, should we keep it?